.github/workflows/ci.yml

```diff
@@ -0,0 +1,208 @@
+name: CI
+
+# What about CI? For the commits?
+on:
+  push:
+    branches:
+      - '*'
+      - '*/*'
+      - '**'
+    tags-ignore:
+      - '**'
+    paths-ignore:
```

Comment:
This action will fail as it will try to execute everything on the main folder. 

You can add the following to limit this workflow to run only when a backend code changes:

```
on:
  push:
    paths:
      - "backend/**"
      - ".github/workflows/ci.yml"
```

Reply:
Don't worry too much about the workflows.  We have plans to overhaul and bring this repo closer to the QuickStart.